go/libkb/id_table.go

```diff
@@ -990,10 +990,24 @@ func (s *WalletStellarChainLink) VerifyReverseSig(_ ComputedKeyFamily) (err erro
 }
 
 func (s *WalletStellarChainLink) Display(m MetaContext, ui IdentifyUI) error {
+	// First get an up to date user card, since hiding the Stellar address affects it.
+	_ = m.G().CardCache().Delete(s.GetUID())
```

Comment:
This seems to mean that every time a user looks at a profile, we delete the user card in the cache?  Am I reading that correctly?

Reply:
@patrickxb Yes, that's right, if they have a Stellar proof.

This is to avoid something like a race where the user hides their wallet and we get back to Display() because of receiving userChange before we've received the new user card. If we had a way to reprocess proofs whenever we get that new user card, I think we could do that instead..

Reply:
I think this destroys the usefulness of the cache for all these users.

What about sending `user.identify_change` gregor msg from the server when they hide/unhide stellar?  And I think adding CardCache().Delete(u) into libkb/globals.go BustLocalUserCache() would be a good idea.

Then I don't think you need this Delete here.

Reply:
@patrickxb Thanks! I was already sending that gregor message in the kbweb PR, so I just made the CardCache move change and it seems to work.